### ISSUE TITLE
Accessibility: unlabeled buttons

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#26]
**Describe the bug**
The buttons in the screenshots are missing labels for screen readers like Talkback on Android.

**To Reproduce**
1. Open system settings > Accessibility > Talkback and toggle it on.
2. Open thunder
3. Tap the icons highlighted in the screenshots with a finger

**Expected behavior**
The buttons should be labeled for example "sort by", "upvote" (or "upvoted" if the post has been upvoted), "downvote" (or "downvoted" if the post has been downvoted), "save" (or "saved" if the post has been saved), etc

**Screenshots**
![Screenshot_20230620-102631_thunder](https://github.com/hjiangsu/thunder/assets/116814522/706ae096-c1f5-46ed-b196-74a3392064fd)
![Screenshot_20230620-102640_thunder](https://github.com/hjiangsu/thunder/assets/116814522/344ae85d-0222-41c9-aada-972ec7d240d5)
![Screenshot_20230620-102644_thunder](https://github.com/hjiangsu/thunder/assets/116814522/df094720-1d40-4322-b9f6-f9e5c4ed827b)
![Screenshot_20230620-102649_thunder](https://github.com/hjiangsu/thunder/assets/116814522/81cc0fd9-1a8c-4e0c-9f74-4a2634b00fb4)
![Screenshot_20230620-102716_thunder](https://github.com/hjiangsu/thunder/assets/116814522/53daadcd-6e5e-41a3-b8fa-0a37d6568ded)
![Screenshot_20230620-102719_thunder](https://github.com/hjiangsu/thunder/assets/116814522/b14fbde4-9fcc-48f5-8e17-8e12f8c5e7c7)
![Screenshot_20230620-102723_thunder](https://github.com/hjiangsu/thunder/assets/116814522/dae5e9be-6ae5-49ad-bae8-8a9e89311f3e)
![Screenshot_20230620-102813_thunder](https://github.com/hjiangsu/thunder/assets/116814522/82c2b5da-15ac-4e5f-89eb-196ea794996b)

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] Motorola One 5G Ace
 - OS: [e.g. iOS8.1] Android 11
 - Browser [e.g. stock browser, safari] N/A
 - Version [e.g. 22] browser: N/A, App v0.1.1-ALPHA

**Additional context**

**Comment by hjiangsu** [comment]
Just wanted to give a quick update - I'm currently working on this issue!

I don't have much knowledge on proper conventions for adding accessibility, so it would be great if you could provide feedback in the future once this has been released!

**Comment by shortwavesurfer2009** [comment]
> Just wanted to give a quick update - I'm currently working on this issue!
> 
> I don't have much knowledge on proper conventions for adding accessibility, so it would be great if you could provide feedback in the future once this has been released!

Absolutely. The biggest thing is use as few words to describe as needed. For example instead of a label that says "open sort options" use "sort by". Instead of a label that reads "go to settings" use "settings" instead. Fewest words possible that gets the point of the button across.

**Comment by hjiangsu** [comment]
Alright, applied the feedback from this issue - to be released in the upcoming release! Once that's out it, would be great if you could double check and provide any more feedback!

**Comment by shortwavesurfer2009** [comment]
I just tried the prerelease version found here https://github.com/hjiangsu/thunder/releases/tag/v0.2.1-prerelease%2B5 and this version fixes everything here. Closing as completed

**Comment by hjiangsu** [comment]
Thanks for trying it out! I'll keep this open for now for visibility purposes (in case someone else wants to open an issue regarding this) since its not released as a main alpha version yet, just the pre-release

**Comment by hjiangsu** [comment]
Closing this now as this has been released :D
